openapi3/schema.go

```diff
@@ -1643,7 +1644,10 @@ func (schema *Schema) visitJSONNumber(settings *schemaValidationSettings, value
 	if v := schema.MultipleOf; v != nil {
 		// "A numeric instance is valid only if division by this keyword's
 		//    value results in an integer."
-		if bigFloat := big.NewFloat(value / *v); !bigFloat.IsInt() {
+		numParsed, _ := decimal128.Parse(fmt.Sprintf("%.10f", value))
```

Comment:
Going higher, example %.15f here will break test case like `19628.87`
As using %.15f turns this to 19628.869999999989.... causing non zero remainder later. I think it's okay to lose a little precision here to have precision in the remainder.